gr-azure-software-radio/python/blob_sink.py

```diff
@@ -38,13 +41,15 @@ class BlobSink(gr.sync_block):
     Blob Block Length: How many items to write out at once to a block in the blob. Note that
         block sizes where block_len*itemsize is greater than 4MiB will enable the use of
         high throughput block transfers. Must be an integer number of items.
+    Retry Total: Total number of Azure API retries to allow before throwing an exception. Higher
+        numbers make the block more resilient to intermittent failures, lower numbers help to
+        more quickly iteratively debug authentication and permissions issues.
     """
     # pylint: disable=too-many-arguments, too-many-instance-attributes, arguments-differ, abstract-method
 
     def __init__(self, authentication_method: str = "default", connection_str: str = None,
                  url: str = None, container_name: str = None, blob_name: str = None,
-                 block_len: int = 500000, queue_size: int = 4):
-        # pylint: disable=no-member
```

Comment:
do we need the pylint: disable=no-member?

Reply:
I didn't see any pylint errors in the latest build with it gone. Think we still need it?

Reply:
we don't need it then :)